src/components/SearchBar/StyledSearchResult.js

```diff
@@ -0,0 +1,51 @@
+import styled from "styled-components";
+import SearchResult from "./SearchResult";
+
+export default styled(SearchResult)`
+  display: ${props => (props.show ? `block` : `none`)};
+  max-height: 80vh;
+  overflow: scroll;
+  -webkit-overflow-scrolling: touch;
+  position: absolute;
+  z-index: 2;
+  right: 44%;
+  top: 100%;
+  margin-top: 0.25em;
+  width: 100vw;
+  max-width: 30em;
+  padding: 1em;
+  border-radius: 2px;
+  background-color: ${props =>
+    props.mode === "light"
+      ? "rgba(242, 241, 239, 1)"
+      : "var(--color-headerBg)"};
+
+  .HitCount {
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  .Hits {
+    ul {
+      list-style: none;
+      margin-left: 0;
+    }
+
+    li.ais-Hits-item {
+      margin-bottom: 1em;
+
+      a {
+        color: ${props => (props.mode === "light" ? "black" : "white")};
```

Comment:
We should be using the utils theme.js file to control this. Setting it up any other way will cause flashing problems. The css variables are loaded when the page is so there should be no problem here